### ISSUE TITLE
What to do about Norway?

### ISSUE THREAD

**Issue by konklone** [unitedstates/licensing#31]
http://lists.ibiblio.org/pipermail/cc-community/2014-October/008866.html

> In Norway, by law, if you publicly perform, or reproduce, materials that are in the Public Domain (because the creator dedicated his/her work to the public domain), you'll have to pay a /levy/ to one the
> national collection agencies that oversees that particular art form (e.g.GRAMO for music, KOPINOR for text).
> 
> Why it may sound bizarre to you, the fact is that Norwegian collection agencies has successfully managed to get it into the law a provision that entails that dedicating anything to the public domain anywhere in the world effectively transfer management of its rights /in Norway/ to the collection agency.  They can legally do this, because dedicating stuff to the public domain means that the creator is no longer in a position to manage its copyright.

And he's apparently validated this with the Norwegian government:

> I think people should be encouraged to dual license: CC-BY for Norway, and CC0 for the rest of the world.  I now have a letter from the department of culture confirming that the collection agencies are entitled to a levy if one uses works in the public domain, but not for those under copyright with a free culture license (i.e. CC-BY).

And in http://lists.ibiblio.org/pipermail/cc-community/2014-October/008870.html -

> When someone plays recorded music that is under CC0 in a public space (a park, a shopping mall, a cafe, etc.). GRAMO have roving inspectors that will spot such occurences, and demand the levy to be paid.  And they will take you to court if you don't pay, and you'll lose.

If true, that's wildly obnoxious. @tvol, you have any thoughts on this?

**Comment by JoshData** [comment]
As I reflect, I think this is Norway's problem. The OP suggests everyone write a license that treats Norway differently. If Norway wants to not have a true public domain, that's their policy to be had.

**Comment by tvol** [comment]
Yeah, what a horrible policy. I guess one mitigating factor is that it's constrained to music and text collecting societies (and of course just in Norway). And, it seems like the music collecting societies have been more active in actually trying to enforce this. I'm not sure if there's any legal hack that could get around this other than using a license instead of CC0, but the CC legal team is looking into it. I don't think it should change our recommendations to USG to use CC0. Maybe the existence of this strange and unfortunate policy will just have to be marked with an asterisk--although it's definitely something to keep an eye on especially if others begin to try to replicate it.
